Command new (#107)
intel/he-toolkit: kit/README.md

hekit remove hexl 1.2.3
```

> The command new can be used to create a new application. When it is executed

Reviewer: ```suggestion
The command `new` can be used to create a new project. When it is executed
```

with the option `-–based-on`, it will create a copy of the base project,
keeping its directory structure.
```bash
hekit new my-secure-query --based-on secure-query
```

> However, when the command is executed without `-–based-on`.

Reviewer: ```suggestion
However, when the command is executed without `-–based-on`
```

```bash
hekit new example
```

It will create the following directory structure:
```bash
Projects
└── example
      |- CMakeLists.txt
      |- README.md
      └── include
            └── example.h
      └── recipes
            └── example.toml
      └── src
            └── example.cpp
```

> And the following actions should be completed to build the new application:

Reviewer: ```suggestion
The following actions should be completed to build the new application:
```

> * Open the toml file inside the recipes directory and replace `-DFLAG=TBD` with the required flags

Reviewer: ```suggestion
* Open the toml file inside the recipes directory and replace `-DFLAG=TBD` with the desired CMake flags for your project.
```

> * Open CMakeLists.txt and uncomment the statements for find_package and target_link_libraries of the required library. If other dependencies are needed, for instance Threads, the file must be updated to include and compile them.

Reviewer: ```suggestion
* Open `CMakeLists.txt` and uncomment the statements for `find_package` and `target_link_libraries` of the required library. If other dependencies are needed, for instance `Threads`, the file must be updated to include and compile them.
```

> * Add the code of the new application in the .cpp and .h files created by the command

Reviewer: ```suggestion
* Add and/or write the code of the new application in the `.cpp` and `.h` files created by the command.
```
